### ISSUE TITLE
Sample test project

### ISSUE THREAD

**Issue by cristianmad** [TheKeithStewart/angular-esri-components#16]
Hi,
 
Is there a sample project using the component? (or maybe some more detailed instructions)

Thanks

**Comment by TheKeithStewart** [comment]
Hi @cristianmad,

Yes.  https://github.com/tomwayson/esri-angular-cli-example is using this library.

Let me know if you are having difficulty with any particular feature.

**Comment by cristianmad** [comment]
@TheKeithStewart Thank you for the quick reply!

The link provided by you is exactly what I was looking for! 👍 I think that it may be useful having the link on your component's main page (unless it is already there :) )
I will go over the example and let you know if I run into any issues.

Thanks again, 
Cristian

**Comment by TheKeithStewart** [comment]
Good point, @cristianmad.  I'll add a link to the example project in the README.md.
